src/coreclr/vm/codeman.cpp

```diff
@@ -3749,7 +3928,7 @@ BOOL EEJitManager::GetBoundariesAndVars(
         SUPPORTS_DAC;
     } CONTRACTL_END;
 
-    CodeHeader * pHdr = GetCodeHeaderFromDebugInfoRequest(request);
+    TCodeHeader * pHdr = GetCodeHeaderFromDebugInfoRequest<TCodeHeader>(request);
```

Comment:
`GetBoundariesAndVarsWorker` does not have to be template if we can change the argument for `GetBoundariesAndVarsWorker` to be `PTR_BYTE pDebugInfo`, move the `GetCodeHeaderFromDebugInfoRequest` to the caller.

Reply:
Nice, I did the same to the GetRichDebugInfoWorker

Reply:
Done